### ISSUE TITLE
I have the current layout (disabled) and i see this

### ISSUE THREAD

**Issue by MesaBlack** [lightbeam24/CustomTube#162]
![image](https://github.com/lightbeam24/CustomTube/assets/132141735/f414b5f6-734e-41ac-997c-96427a189ea9)

**Comment by lightbeam24** [comment]
Yeah there appears to be some code that still applies to the disabled layout when it's not supposed to. Will fix eventually, but it's not a high priority.
